script/Ciel/Day 10/10_01_CIEL10_4_3.txt

```diff
@@ -387,92 +414,105 @@ Yeah... Setting that aside―――
 [sha:b24af8a27f943b551ad92a9656093b248bd4246a]{
 -- Page 269, Offset 27797.
 -- 「ですがアルクェイドとの協力だけは出来ません。
-"But cooperating with Arcueid is the one thing I cannot agree with.
+"However, cooperating with Arcueid is the one thing I cannot allow.
 }
 [sha:5658a540b35338c9add76455fc508c1ca27f9f4c]{
 -- Page 269, Offset 27798.
 -- この際ですからはっきりと忠告……いえ、警告しておきましょう。現在、この街で最も危ういのは彼女です。それは彼女が吸血鬼だから、という事ではありません。
-This is a good opportunity for me to advise you... no, to warn you. As we speak, there is none more dangerous in this city than her. This is not due to the fact that she is a vampire.
+If you're going to be involved,
+ then let me give you some advice... no, let me give you a warning.
+ As things stand, that woman is the most dangerous creature in this city.
+ That's not just because she's a vampire.
 }
 [sha:b6812623c84b0c45a65a7fe689cc2f4453d0225a]{
 -- Page 269, Offset 27799.
 -- 今の彼女はかつてないほど異常な状態です。下手に刺激すれば炸裂する爆弾と言ってもいい」
-She is currently behaving more abnormally than at any time in the past. You could liken her to a bomb that will violently explode if badly handled."
+The way she's behaving right now is utterly unprecedented.
+ She's like a bomb that could go off in your face if mishandled."
```

Comment:
Something about the metaphor feels a bit uninspired to me.
>In fact, she's practically like a bomb that could go off at the slightest provocation.

Reply:
That feels a bit more muddled to me tbh.

Reply:
Honestly, I don't feel too strongly about it now that I look at it.
Though I do think my suggestion is more evocative.